builder.json

```diff
@@ -66,6 +66,14 @@
         "al2012": {
             "!packages": [],
             "!imports": [ "node-14" ]
+        },
+        "macos": {
+            "_comment": "Mac OS (arm64) has only experimental support for Node v15 and earlier, so stick to Node v16, see Node v15 docs: https://github.com/nodejs/node/blob/v15.x/BUILDING.md#platform-list",
```

Comment:
trivial: move the comment within the `arm64` architectures, since it's specific to arm64 Mac.